### DISCUSSION TITLE
Anaconda.org returning 405 error for jaxlib uploads

### DISCUSSION THREAD

**Discussion by jaimergp** [conda/infra#592] [Q&A] [answered]
Originally reported by @ngam

```
[ERROR] ('Method Not Allowed: Specified method is invalid for this resource. ([POST] https://api.anaconda.org/commit/cf-staging/jaxlib/0.3.15/linux-64/jaxlib-0.3.15-cuda112py37heb223d1_201.tar.bz2 -> 405)', 405)
```

See https://github.com/conda-forge/jaxlib-feedstock/issues/136

**Comment by chrisburr** [comment]
This is also being seen on other feedstocks: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=553498&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d

**Reply by chrisburr** [reply]
It seems to be related to the file size as uploading small packages works but larger ones fail. To reproduce it against my personal channel:

```bash
$ curl -LO https://anaconda.org/cf-staging/dart-sdk/2.17.6/download/win-64/dart-sdk-2.17.6-h57928b3_2.tar.bz2
$ anaconda upload -u chrisburr -l test dart-sdk-2.17.6-h57928b3_2.tar.bz2
```

Results in:

```
Uploading file "chrisburr/dart-sdk/2.17.6/win-64/dart-sdk-2.17.6-h57928b3_2.tar.bz2"
 uploaded 176649 of 176649Kb: 100.00% ETA: 0.0 minutes
[ERROR] ('Method Not Allowed: Specified method is invalid for this resource. ([POST] https://api.anaconda.org/commit/chrisburr/dart-sdk/2.17.6/win-64/dart-sdk-2.17.6-h57928b3_2.tar.bz2 -> 405)', 405)
```

**Reply by chrisburr** [reply]
I wondered if it was because conda-forge was still using `anaconda-client` `1.8.0` so I updated to `1.10.0` and see the same issue.

**Reply by chrisburr** [reply]
It seems like this is now working.

**Comment by jaimergp** [comment, marked as answer]
Reported here: https://anaconda.statuspage.io/incidents/h8mw4m6078l3
